config/routes/group.rb

```diff
@@ -18,8 +18,10 @@
     resources :members, only: %i[create destroy index new update]
 
     resources :bots, only: %i[create destroy index new] do
+      get :new_destroy
```

Comment:
As per my comments above this and the new_revoke routes would need to be changed

Reply:
Changed in 44216f0f3d059fa28a6e3abbd5f78ff3419ceaf8